source/form.html

```diff
@@ -3,7 +3,100 @@
+							<label class="filter__label">
+								<input class="filter__input" type="radio" name="type" value="toy">Детская игрушка
+							</label>
+						</li>
+					</ul>
+				</fieldset>
+				<fieldset class="form__filter-group">
+					<legend class="form__filter-title">Цвет</legend>
+					<ul class="form__filter-list">
+						<li class="form__filter-item filter">
+							<label class="filter__label">
+								<input class="filter__input" type="checkbox" name="filter-color" value="white">Белый
+							</label>
+						</li>
+						<li class="form__filter-item filter">
+							<label class="filter__label">
+								<input class="filter__input" type="checkbox" name="filter-color" value="grey">Серый
+							</label>
+						</li>
+						<li class="form__filter-item filter">
+							<label class="filter__label">
+								<input class="filter__input" type="checkbox" name="filter-color" value="tiffany">Тиффани
+							</label>
+						</li>
+						<li class="form__filter-item filter">
+							<label class="filter__label">
+								<input class="filter__input" type="checkbox" name="filter-color" value="black">Черный
+							</label>
+						</li>
+						<li class="form__filter-item filter">
+							<label class="filter__label">
+								<input class="filter__input" type="checkbox" name="filter-color" value="pink">Розовый
+							</label>
+						</li>
+						<li class="form__filter-item filter">
+							<label class="filter__label">
+								<input class="filter__input" type="checkbox" name="filter-color" value="orange">Оранжевый
+							</label>
+						</li>
+					</ul>
+				</fieldset>
+				<fieldset class="form__filter-group">
+					<legend class="form__filter-title">ФИО</legend>
+					<label class="form__filter-label">Имя:
+						<input class="form__filter-input" name="name" type="text" placeholder="Введите ваше имя*">
+					</label>
+					<label class="form__filter-label">Фамилия:
+						<input class="form__filter-input" name="surname" type="text" placeholder="Укажите фамилию*">
+					</label>
+					<label class="form__filter-label">Отчество:
+						<input class="form__filter-input" name="father-name" type="text" placeholder="Отчество, если желаете">
+					</label>
+				</fieldset>
+				<fieldset class="form__filter-group">
+					<legend class="form__filter-title">Тел</legend>
+					<label class="form__filter-label">
+						<span class="visually-hidden">Тел</span>
+						<input class="form__filter-input" name="phone" type="tel" placeholder="+7 ХХХ ХХХ ХХ ХХ*">
+					</label>
+				</fieldset>
+				<fieldset class="form__filter-group">
+					<legend class="form__filter-title">E-mail</legend>
+					<label class="form__filter-label">
+						<span class="visually-hidden">E-mail</span>
+						<input class="form__filter-input" name="email" type="email" placeholder="Ваш Email*">
+					</label>
+				</fieldset>
+				<div class="form__filter-group">
+					<h3 class="form__filter-title">Доп</h3>
+					<span class="visually-hidden">Пожелания</span>
```

Comment:
А где же лэйбл?)
И скрытую дописку можно прямо в заг… в лейбл добавить:

```suggestion
					<label class="form__filter-title">Доп<span class="visually-hidden">олнительные пожелания:</span></label>
```
Только про `for` не забудь, и `id` у `textarea` 👀